modules/tinymce/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The default value for `end_container_on_empty_block` option has been changed to `'blockquote'` #TINY-6559
 
 ### Fixed
+- Images were not showing as selected when selecting all content #TINY-5947
```

Comment:
This also changes how images look while selecting around contents. Like `a[a<img>b]b` so there should be a changelog item about that change. It now shows a proper background for those as it should so it feels like a bug fix to me.

- [ ] Add changelog item about background selection change for images

Reply:
I did consider this originally, however I ended up not including it. I don't think this should be a separate changelog though and instead should just be a single changelog entry so I'll look at updating the existing one instead.

Reply:
Have reworked the changelog to try and address this. How does that look?